package.json

```diff
@@ -54,7 +54,7 @@
   },
   "devDependencies": {
     "babel-eslint": "8.2.1",
-    "css-loader": "0.28.11",
+    "css-loader": "1.0.0",
```

Comment:
@karnthis I couldn't test it , with this version `css-loader` is not support `minimize` option
And after this css files are not minified.
I'll find something how to fix that

Has been created a ticket for that https://github.com/acacode/flamebird/issues/52